### PR TITLE
Fix gallery page

### LINKED ISSUE #781: All Maps not displayed
**Description**
@jywarren 
I guess with https://github.com/publiclab/mapknitter/commit/8d2bb866f0cf191d9af3d238b96b9f95fb75bdc8 the author is not showing up.

I think we should revert it back.

The error for reference:
Selection_195

### PR DESCRIPTION
**Pull request by cesswairimu**
Fixes #781
* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/mapknitter-reviewers` for help, in a comment below

### CODE REVIEW

**Comment by cesswairimu** [comment]
screencapture-localhost-3000-gallery-2019-07-02-21_27_22

**Comment by jywarren** [comment]
I'll restart this!

**Comment by cesswairimu** [comment]
@publiclab/mapknitter-reviewers please review

**Comment by jywarren** [comment]
@cesswairimu I'm seeing the same issue in #782 -- I think `codecov/patch — 16.66% of diff hit (target 71.41%)` means we need to add tests for new code so the coverage of the new code is at least as good as the coverage of the rest of the codebase.

File: app/controllers/front_ui_controller.rb [hunk 1]
```diff
@@ -18,17 +18,15 @@ def all_maps
...
-    if current_location.present?
-      lat = session[:lat]
-      lon = session[:lon]
-      @nearby_maps = Map.maps_nearby(lat: lat, lon: lon, dist: 10)
-                        .page(params[:maps])
-                        .per_page(12)
-    end
-
-    @all_mappers = Map.featured_authors.paginate(page: params[:mappers], per_page: 12)
+    return unless current_location.present?
```
Comment (jywarren) [inline app/controllers/front_ui_controller.rb]: maybe we need to test this too?

Comment (cesswairimu) [reply app/controllers/front_ui_controller.rb]: Oh yeah, makes sense...adding the test. Thanks Jeff

**Comment by jywarren** [comment]
Hooray!!! Merging!

**Pull request merged**